codegen-core/src/main/kotlin/software/amazon/smithy/rust/codegen/core/smithy/generators/GenericsGenerator.kt

```diff
@@ -3,12 +3,12 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-package software.amazon.smithy.rust.codegen.client.smithy.generators
+package software.amazon.smithy.rust.codegen.core.smithy.generators
```

Comment:
I think this class belongs in `rustlang` with the code that uses it.

Reply:
Done in [Move GenericsGenerator into rustlang and rename it to RustGenerics](https://github.com/awslabs/smithy-rs/pull/1762/commits/d6385d96267cd8e564fbc2037c318560afe64ad6). I also renamed it to `RustGenerics`.